### PR TITLE
Remove old, unused scripts.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
These are mostly old (pre Katello 2.0) scripts that are never used anymore or exist confusingly (e.g. katello_remove here is NOT the one we deploy with our RPMs)

**Comment by thomasmckay** [comment]
ack
Lots of memories but not much else!

File: scripts/iso/install_packages [hunk 1]
```diff
@@ -1,192 +0,0 @@
...
-# script exists in a directory which is a valid repo. It should have
-# a Packages and repository subdirectorties
-#
-# After this command is run, the user will need to run the
-# the katello-installer script in order to set up the installation.
-#
-# Copyright 2013-2014 Red Hat, Inc.
-#
-#
-import os
```
Comment (jlsherrill) [inline scripts/iso/install_packages]: we may want to keep this file around

**Comment by ehelms** [comment]
@jlsherrill udpated to keep the iso folder for now

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**